lib/chef/guard_interpreter/default_guard_interpreter.rb

```diff
@@ -33,7 +33,9 @@ def initialize(command, opts)
       public
 
       def evaluate
-        shell_out(@command, @command_opts).status.success?
+        result = shell_out(@command, @command_opts)
+        Chef::Log.warn "Command failed: #{result.stderr}" unless result.status.success?
```

Comment:
but "failure" here in terms of exit != 0 is a normal condition though isn't it so we shouldn't warn?  we could debug log the stderr and stdout and it would seem to be a decent idea to actually always do that.

its also odd that we rescue a timeout below and then swallow the exception and return false.  if we did that for a reason it should probably be noted in a comment as to why that got done, otherwise i'd think that any exception that raised by shell_out should propagate upwards and kill the chef run.